MagickCore/nt-base.h

```diff
@@ -29,7 +29,11 @@ extern "C" {
 
 #define WIN32_LEAN_AND_MEAN
 #define VC_EXTRALEAN
+#ifndef __MINGW32__
 #define _CRT_SECURE_NO_DEPRECATE  1
```

Comment:
Why is this added?

Reply:
Otherwise you get million "_CRT_SECURE_NO_DEPRECATE redefined" warnings in the build. I suppose mingw-w64 defines this by default

Reply:
I have applied these changes with a separate commit.